cmd/proxy/Dockerfile

```diff
@@ -5,10 +5,11 @@ WORKDIR $GOPATH/src/github.com/gomods/athens
 
 COPY . .
 
-RUN CGO_ENABLED=0 go build -mod=vendor -o /bin/app ./cmd/proxy
+RUN CGO_ENABLED=0 GO111MODULE=on go build -mod=vendor -o /bin/app ./cmd/proxy
+RUN ./scripts/create_default_config.sh
 
 ENV GO_ENV=production
 
 EXPOSE 3000
 
-ENTRYPOINT ["/bin/app"]
+ENTRYPOINT /bin/app -config_file=config.toml
```

Comment:
Shouldn't this be `ENTRYPOINT ["/bin/app", "-config_file=config.toml"]` ?

Reply:
they both work :)